components/dot_menu/dot_menu.tsx

```diff
@@ -441,6 +465,24 @@ export class DotMenuClass extends React.PureComponent<Props, State> {
                             onClick={this.handleAddReactionMenuItemActivated}
                         />
                     </ChannelPermissionGate>
+                    <Menu.ItemAction
+                        id={`follow_post_thread_${this.props.post.id}`}
+                        show={(
+                            this.props.isCollapsedThreadsEnabled &&
+                                (
+                                    this.props.location === Locations.RHS_ROOT ||
+                                    this.props.location === Locations.RHS_COMMENT
+                                )
+                        )}
+                        {...(this.props.isFollowingThread) ? {
+                            text: this.props.threadReplyCount ? Utils.localizeMessage('threading.threadMenu.unfollow', 'Unfollow thread') : Utils.localizeMessage('threading.threadMenu.unfollowMessage', 'Unfollow message'),
+                            extraText: Utils.localizeMessage('threading.threadMenu.unfollowExtra', 'You won’t be notified about replies'),
```

Comment:
Cool trick with the spread :+1:

Reply:
@calebroseland deserves the credit for that on! I just saw it, liked it, and stole it :smile: